content/docs/integrations/external-dns-cert-manager.md

```diff
@@ -1,21 +1,28 @@
 ---
-title: ExternalDNS & Cert Manager
+title: ExternalDNS and cert-manager
 weight: 520
-description: Use kgateway with External DNS and Cert Manager. 
 ---
 
-[ExternalDNS](https://github.com/kubernetes-sigs/external-dns) and [Cert Manager](https://github.com/cert-manager/cert-manager) are two well known integrations within the Kubernetes ecosystem that can be used in conjunction to automate the creation of TLS certificates. 
+[ExternalDNS](https://github.com/kubernetes-sigs/external-dns) and [cert-manager](https://github.com/cert-manager/cert-manager) are two well-known integrations within the Kubernetes ecosystem that can be used in conjunction to automate the creation of TLS certificates. 
+
+* **ExternalDNS**: Instead of manually editing your domain in your DNS provider to add load balancer IP addresses, you can use ExternalDNS to dynamically set up and control DNS records for discovered gateway and HTTP resources. When you define a hostname in an HTTPRoute resource, ExternalDNS uses the external address that is assigned to the gateway's load balancer service that serves this hostname, and uses this information to create a DNS record in the DNS provider that you configured.
+* **cert-manager**: You can then use cert-manager to quickly and programmatically generate certificates for your domain, and store them in a Kubernetes secret. By adding this certificates secret to the HTTPRoute on the gateway for your domain, you can secure it for HTTPS traffic.
 
 ## Before you begin 
 
 {{< reuse "docs/snippets/prereq.md" >}}
 
 ## Set up ExternalDNS
-ExternalDNS is used to dynamically set up and control DNS records for discovered gateway and HTTP resources. When you create a gateway or HTTP resource, and you define a hostname, External DNS uses the external address that is assigned to the gateway's load balancer service that serves this hostname, and uses this information to create a DNS record in the DNS provider that you configured. 
 
-You can later use Cert Manager to create TLS certificates for this hostname so that you can serve HTTPS traffic on your gateway. 
+Use ExternalDNS to dynamically set up and control DNS records for discovered gateway and HTTP resources. When you define a hostname in an HTTPRoute resource, ExternalDNS uses the external address that is assigned to the gateway's load balancer service that serves this hostname, and uses this information to create a DNS record in the DNS provider that you configured.
+
+In the next section, you use cert-manager to create TLS certificates for this hostname so that you can serve HTTPS traffic on your gateway. 
```

Comment:
```suggestion
In the next section, you use cert-manager to create TLS certificates for this hostname so that you can serve HTTPS traffic on your Gateway. 
```